src/pages/Transaction/TransactionCell/index.tsx

```diff
@@ -280,15 +280,142 @@ const useParseNftInfo = (cell: Cell) => {
+
+  const { typeHash, tokenId } = (() => {
+    if (cell.cellType !== 'spore_cell' && cell.cellType !== 'did_cell') {
+      return { typeHash: undefined, tokenId: undefined }
+    }
+
+    if (!cell.extraInfo || !('collection' in cell.extraInfo)) {
+      return { typeHash: undefined, tokenId: undefined }
+    }
+
+    return {
+      typeHash: cell.extraInfo.collection?.typeHash,
+      tokenId: cell.extraInfo.tokenId,
+    }
+  })()
+
+  const { data, isLoading } = useQuery(
+    ['nft-item-info', typeHash, tokenId],
+    () => explorerService.api.fetchNFTCollectionItem(typeHash!, tokenId!),
+    {
+      enabled: !!typeHash && !!tokenId && (cell.cellType === 'spore_cell' || cell.cellType === 'did_cell'),
+    },
+  )
+
+  useEffect(() => {
+    if (cell.cellType !== 'spore_cell' && cell.cellType !== 'did_cell') {
+      return
+    }
+
+    if (!cell.extraInfo) {
+      return
+    }
+
+    getSporeImg({ data: cell.extraInfo.data, id: cell.extraInfo.tokenId }).then(res => {
+      setCover(res)
+    })
+  }, [cell])
+
+  return {
+    data: {
+      cover,
+      nftInfo: data,
+    },
+    isLoading,
+  }
+}
+
 export const TransactionCellDetail = ({ cell }: { cell: Cell }) => {
   const { t } = useTranslation()
-  let detailTitle = t('transaction.ckb_capacity')
+  let detailTitle: JSX.Element | string = <span>{t('transaction.ckb_capacity')}</span>
   let detailIcon
   let tooltip: string | ReactNode = ''
   const nftInfo = useParseNftInfo(cell)
+  const { data: dobInfo } = useDOBInfo(cell)
+
+  const lockScript = addressToScript(cell.addressHash)
   const isMultisig = (cell.tags ?? []).find(tag => tag === 'multisig') !== undefined
   const isFiber = (cell.tags ?? []).find(tag => tag === 'fiber') !== undefined
   const isDeployment = (cell.tags ?? []).find(tag => tag === 'deployment') !== undefined
+  const isDob = cell.cellType === 'spore_cell' || cell.cellType === 'did_cell'
+  const isZeroLock = lockScript.codeHash === ZERO_LOCK_CODE_HASH
+
+  const tokenIdStr = `${dobInfo.nftInfo?.standard === 'spore' ? '' : '#'}${formatNftDisplayId(
+    dobInfo.nftInfo?.token_id ?? '',
+    dobInfo.nftInfo?.standard ?? null,
+  )}`
+
+  const taglists = [
+    {
+      tag: 'Dob Protocol',
+      description: 'This cell is related to Dob Protocol.',
+      display: isDob,
+      link: `/script/${scriptDataMap.Spore.codeHashes[0]}/${scriptDataMap.Spore.hashType}`,
+      icon: DobIcon,
+      iconColor: '#FFFEC1',
+    },
+    {
+      tag: 'Deployment',
+      description: 'This cell is related to Deployment.',
+      display: isDeployment,
+      link: '/scripts',
+      icon: DobIcon, // The ui doesn't provide an icon yet, so use another icon for now.
+    },
+    {
+      tag: 'Multisig',
+      description: 'This cell is related to Multisig. It can be used to create a multisig address.',
+      display: isMultisig,
+      link: `/script/${scriptDataMap['secp256k1 / multisig'].codeHashes[0]}/${scriptDataMap['secp256k1 / multisig'].hashType}`,
```

Comment:
Now we have several multisig, so the link should be flexible.
The `code hash` and `hash type` can be decoded from the `address`

Link in the fiber tag should be constructed in the same way.

Reply:
Please check this @PainterPuppets